### PR TITLE
Add FluentHub & Depth

### PR DESCRIPTION
**Pull request by DeveloperWOW64**
Adds [`FluentHub`](https://github.com/fluenthub/fluenthub) and [`Depth`](https://github.com/DepthCDLS) to the users page.

### CODE REVIEW

**Review by juanpicado — APPROVED** [review]
LGTM

File: users.json [hunk 1]
```diff
@@ -75,6 +75,18 @@
     "infoLink": "https://files.community/",
     "pinned": false
   },
+  {
+    "caption": "Depth",
```
Comment (juanpicado) [inline users.json:79]: This is a personal website, doesn't?  I'm not sure if fits o this page usually are projects or companies, any thought @zkochan ?

**Comment by zkochan** [comment]
We are not adding any more users to the website.
Only our sponsors and very well known companies.

If you want to share your stack, you can do that on stackshare: https://stackshare.io/pnpm

File: users.json (hunk 1 above)
Comment (DeveloperWOW64) [reply users.json:79]: > This is a personal website, isn't it?
Not necessarily -- it's going through major refactor changes (not available yet on the website)

**Comment by DeveloperWOW64** [comment]
> We are not adding any more users to the website. 

Then what about [`Files Community`](https://github.com/pnpm/pnpm.github.io/pull/229)?

**Comment by zkochan** [comment]
Looks like already a popular project. Lets keep it.

The issue is that some people started to use the users page as a way to promote their companies and/or projects.

**Pull request closed without merging**